### PR TITLE
build: add more pipelines

### PR DESCRIPTION
**Pull request by yorkie**
Add 'mnist-image-classification' and 'databinding-image-classification' in our Pipeline CI.

### CODE REVIEW

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -1,16 +1,18 @@
 name: pipeline
 on:
+  push:
```
Comment (yorkie) [inline .github/workflows/pipeline.yml:3]: Just for debugging, will remove this before this can get merged.

**Comment by yorkie** [comment]
The Pipeline has been completed at https://github.com/alibaba/pipcook/actions/runs/79003216.

**Comment by utkobe** [comment]
@yorkie  can this one be reviewed and merged

**Comment by yorkie** [comment]
Yep, I have removed the push trigger :)

**Pull request merged**